auto_install/install.sh

```diff
@@ -84,15 +84,29 @@ distro_check() {
   if hash lsb_release 2>/dev/null; then
       PLAT=$(lsb_release -si)
       OSCN=$(lsb_release -sc) # We want this to be trusty xenial or jessie
-  
-      if [[ $PLAT == "Ubuntu" || $PLAT == "Raspbian" || $PLAT == "Debian" ]]; then
-          if [[ $OSCN != "trusty" && $OSCN != "xenial" && $OSCN != "jessie" ]]; then
-              maybeOS_Support
-          fi
+
+      case ${PLAT} in
+          Ubuntu|Raspbian|Debian|Devuan)
```

Comment:
hey @redfast00 , why only here is `Ubuntu|Raspbian|Debian|Devuan` and other places is `Ubuntu|Debian|Devuan`

why don't you do a global var to avoid repeating it?